pipelines/Conversions.yaml

```diff
@@ -32,6 +32,10 @@ tasks:
   # Conversion of visitSummary [visit-level afw.table] to visitTable [instrument-level Parquet]
   makeVisitTable:
     class: lsst.pipe.tasks.postprocess.MakeVisitTableTask
+    config:
+      connections.visitSummaries: visitSummary
```

Comment:
If the goal is to restore the connection name to what it was before lsst/pipe_tasks#741, then please make this `{calexpType}visitSummary`. We get a lot of bugs from importing pipelines that have connections overridden to fixed values; it's better that it always has the templates we expect it to, even if we don't expect to use them.

Reply:
Will do; looks like that will also let me drop these overrides from the `ConversionsForFakes.yaml` since they'll be imported.

Reply:
But `consolidateVisitSummaryWithFakes` is not imported, it's brand-new.

Reply:
Ah, thanks :man_facepalming: .

Reply:
Actually, I think that's okay, I think, because `makeVisitTable` and `makeCcdVisitTable` are not new there, and they're the ones getting the overrides.  `ConsolidateVisitSummaryTask`'s defaults have not been changed upstream, only the defaults of the tasks that previously consumed `{calexpType}visitSummary`.